staging/src/k8s.io/client-go/INSTALL.md

```diff
@@ -17,65 +15,86 @@ The next time you `go build`, `go test`, or `go run` your project,
 and detailed dependency version info will be added to your `go.mod` file
 (or you can also run `go mod tidy` to do this directly).
 
-This assumes you are using go modules with go 1.11+.
-If you get a message like `cannot use path@version syntax in GOPATH mode`,
-see the instructions for [enabling go modules](#enabling-go-modules).
+## Using a specific version
 
-## Dependency management for the serious (or reluctant) user
+If you want to use a particular version of the `k8s.io/client-go` library,
+you can indicate which version of `client-go` your project requires:
 
-Reasons why you might need to use a dependency management system:
-* You use a dependency that client-go also uses, and don't want two copies of
-  the dependency compiled into your application. For some dependencies with
-  singletons or global inits (e.g. `glog`) this wouldn't even compile...
-* You want to lock in a particular version (so you don't have to change your
-  code every time we change a public interface).
-* You want your install to be reproducible. For example, for your CI system or
-  for new team members.
+- If you are using Kubernetes versions >= `v1.17.0`, use a corresponding `v0.x.y` tag.
+  For example, `k8s.io/client-go@v0.20.4` corresponds to Kubernetes `v1.20.4`:
 
-### Enabling go modules
+```sh
+go get k8s.io/client-go@v0.20.4
+```
 
-Dependency management tools are built into go 1.11+ in the form of [go modules](https://github.com/golang/go/wiki/Modules).
-These are used by the main Kubernetes repo (>= `v1.15.0`) and `client-go` (>= `kubernetes-1.15.0`) to manage dependencies.
-If you are using go 1.11 or 1.12 and are working with a project located within `$GOPATH`,
-you must opt into using go modules:
+- If you are using Kubernetes versions < `v1.17.0`, use a corresponding `kubernetes-1.x.y` tag.
+  For example, `k8s.io/client-go@kubernetes-1.16.3` corresponds to Kubernetes `v1.16.3`:
 
 ```sh
-export GO111MODULE=on
+go get k8s.io/client-go@kubernetes-1.16.3
 ```
 
-Ensure your project has a `go.mod` file defined at the root of your project.
-If you do not already have one, `go mod init` will create one for you:
+You can now import and use the `k8s.io/client-go` APIs in your project.
+The next time you `go build`, `go test`, or `go run` your project,
+`k8s.io/client-go` and its dependencies will be downloaded (if needed),
+and detailed dependency version info will be added to your `go.mod` file
+(or you can also run `go mod tidy` to do this directly).
```

Comment:
nit: this text is under both `Using the latest version` and `Using a specific version`. Is that intentional/can we remove one of them?

Reply:
interesting... it was in two places in the doc previously as well... I think I'll merge and let a deduplication be a follow-up.